src/virt.c

```diff
@@ -1143,6 +1140,18 @@ static int refresh_lists(struct lv_read_instance *inst) {
         goto cont;
       }
 
+      status = virDomainGetInfo(dom, &info);
+      if (status != 0) {
+        ERROR(PLUGIN_NAME " plugin: virDomainGetInfo failed with status %i.",
```

Comment:
Should this really be logged as an error? I can imagine it happens quite often that a domain is stopped or disappears between the lookup above and this call. Perhaps a warning is more appropriate?